contracts/RToken.sol

```diff
@@ -1,44 +1,39 @@
 pragma solidity ^0.5.8;
 pragma experimental ABIEncoderV2;
 
+import {Storage} from "./Storage.sol";
+import {Structs} from "./Storage.sol";
+import {Proxiable} from "./Proxiable.sol";
+import {LibraryLock} from "./LibraryLock.sol";
 import {SafeMath} from "openzeppelin-solidity/contracts/math/SafeMath.sol";
-import {Ownable} from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
```

Comment:
> Ownable.sol and ReentrancyGuard.sol each have their own "Private" variable declarations when using the imported contracts from OpenZeppelin. Since all the variables are listed as "Public" in Storage.sol, this is actually creating two separate variables, or at least it's unclear which is being used. Regardless, I copied the code from these two contracts to import them locally

Needed to strip out the variables from the OZ contract.

Reply:
I will leave this open, and I will consult a couple of ideas from others and look around examples... I will give myself a bit more time before giving up on this one.